src/main/webapp/category.html

```diff
@@ -13,13 +21,16 @@
       <span class="menu"><a href="/category.html?category=breakfast" class="menunav">Breakfast</a></span>
       <span class="menu"><a href="/category.html?category=lunch" class="menunav">Lunch</a></span>
       <span class="menu"><a href="/category.html?category=dinner" class="menunav">Dinner</a></span>
-      <span class="menu"><a href="recipe.html" class="menunav">Random recipes</a></span>
+      <span class="menu"><a href="random_recipe.html" class="menunav">Random recipes</a></span>
```

Comment:
I was under the impression that random recipes were going to go on index.html?

In that case you don't need `random_recipe.html`.